parsers/NTESMO.py

```diff
@@ -95,11 +96,24 @@ def _scan_daily_report_urls(
         for a in soup.find_all("a", href=True):
             if a["href"].startswith("https://ntesmo.com.au/__data/assets/excel_doc/"):
                 timestamp = a.find("div", {"class": "smp-tiles-article__title"}).text
-                dt = (
-                    datetime.strptime(timestamp, "%d %B %Y")
-                    .replace(tzinfo=AUSTRALIA_TZ)
-                    .date()
-                )
+
+                try:
+                    date_time = datetime.strptime(timestamp, "%d %B %Y")
+                except ValueError:
+                    # some dates are not formatted properly or will contain extra information
+                    # e.g. "05 April 2018 - Publishing Recommencement", "05 January 2018 - System Notice", "18 June2018"
+                    # so extract individual %d %B %Y groups, potentially separated by 0 or more spaces
+                    match = re.search(r"(\d{2})\s*([a-zA-Z]+)\s*(\d{4})", timestamp)
+                    if not match:
+                        raise
```

Comment:
I thought that here we might as well just re-raise the original `ValueError` from the failed `datetime.strptime(timestamp, "%d %B %Y")` attempt as it already contains a useful error message about the problem